content/02.body.md

```diff
@@ -2,13 +2,65 @@
 
 ### Node degree
 
-![Degree figure](){#fig:degree width="100%"}
+Networks contain information about relationships between entities ("edges between nodes").
+One key aspect of a node (especially in unweighted networks) is its degree, the number of connections a node makes with other nodes in the network.
+Degree is an important metric for differentiating between nodes and it is heavily relied-upon by a number of edge prediction (also known as "link prediction") features (Figure {@fig:degree}E).
+However, reliance on degree may not always be desirable.
+Firstly, degree may be a source of bias in networks.
+Secondly, an over-reliance on degree can lead edge prediction methods to make nonspecific or trivial predictions  that fail to identify novel or insightful relationships.
+We are interested in assessing the effect of node degree on edge prediction methods.
+In this paper, we introduce a permutation-based framework to find edge existence probabilities due to node degree and to quantify the contribution of degree to edge prediction methods.
+
+Most biomedical data networks are imperfect representations of the true set of relationships.
+Real networks often mistakenly include edges that do not exist and exclude edges that do exist.
+How well a network represents the true relationships it attempts to represent depends on a number of factors, especially the methods used to generate the data in the network [@doi:10.1016/j.jprot.2014.01.020; @doi:10.3389/fgene.2015.00260; @doi:10.1038/nbt1116].
+We define "degree bias" as the type of misrepresentation that occurs when the fraction of incorrectly existent/nonexistent relationships depends on degree.
+Inspection bias indicates that entities are not uniformly studied [@doi:10.1038/nature04209], and it is likely to cause degree bias when networks are constructed using hypothesis-driven findings extracted from the literature, as newly-discovered relationships are not randomly sampled from the set of all true relationships.
+The presence of inspection bias in a network means that well-studied entities have a higher fraction of their true relationships represented than less well-studied entities.
```

Comment:
Figure 4A at https://doi.org/10.1016/j.cell.2014.10.050 is a brilliant example of inspection bias.

> (A) Adjacency matrices showing Lit-BM-13 (blue) and HI-II-14 (purple) interactions, with proteins in bins of ∼350 and ordered by number of publications along both axes. The color intensity of each square reflects the total number of interactions for the corresponding bins.
>
> ![](https://ars.els-cdn.com/content/image/1-s2.0-S0092867414014226-gr4.jpg)

We could cite it here or add a sentence:

> For example, the number of publications mentioning a gene has little correlation with its degree in a _systematically-derived_ protein interaction network [Figure 6A in @doi:10.1016/j.cell.2014.10.050]. Therefore, the high correlation between number of publications and degree in low-throughput interaction networks is almost entirely the result of inspection bias. In reality, poorly studied genes have just as many interactions as the genes scientists have preferentially examined.

Reply:
Awesome suggestion!